app/models/article.rb

```diff
@@ -108,7 +108,7 @@ def refresh!
 
   def rot!
     update_attribute(:rotted_at, Time.current)
-    Delayed::Job.enqueue(SendArticleRottenJob.new(self.id, contributors))
+    SendArticleRottenJob.perform_later(self, contributors)
```

Comment:
I prefer passing references rather than objects to the job in case anything changes in the database before the Job can be run. This is why it passed `self.id` and I'm currently updating this method to fetch contributors inside the job.